addons/azureSets/azuresets.lua

```diff
@@ -94,8 +104,7 @@ function set_spells_from_spellset(spellset, setPhase)
 
                 windower.ffxi.remove_blue_magic_spell(slotToRemove)
                 --log('Removed spell: '..v..' at #'..slotToRemove)
```

Comment:
I left the debug messages in, but commented for future use. I considered adding a "debug" mode to display these log messages to the user if they choose, but I didn't end up implementing it because it wouldn't be nearly as useful as GearSwap's debug mode...